tools/tenscan/backend/obscuroscan_backend.go

```diff
@@ -30,6 +30,10 @@ func (b *Backend) GetLatestBatch() (*common.BatchHeader, error) {
 	return b.obsClient.BatchHeaderByNumber(nil)
 }
 
+func (b *Backend) GetHealthStatus() (bool, error) {
+	return b.obsClient.HealthStatusOfNode()
+}
```

Comment:
Probably cleaner to maintain the same logic on the obs client and chang the naming on the b* Backend method.

```suggestion
func (b *Backend) GetTenNodeHealthStatus() (bool, error) {
	return b.obsClient.Health()
}
```